05week/spaceTravelToMars.js

```diff
@@ -1,51 +1,89 @@
-'use strict';
+"use strict";
 
-let assert = require('assert');
+let assert = require("assert");
 
 let jobTypes = {
-  pilot: 'MAV',
-  mechanic: 'Repair Ship',
-  commander: 'Main Ship',
-  programmer: 'Any Ship!'
+  pilot: "MAV",
+  mechanic: "Repair Ship",
+  commander: "Main Ship",
+  programmer: "Any Ship!"
 };
 
 // Your code here
+class CrewMember {
+  constructor(name, job, specialSkill, ship) {
+    this.name = name;
+    this.job = job;
+    this.specialSkill = specialSkill;
+    this.ship = null;
+  }
+
+  enterShip(ship) {
+    this.ship = ship;
+    ship.crew.push(this);
+  }
+}
+
+class Ship {
+  constructor(name, type, ability, crew) {
+    this.name = name;
+    this.type = type;
+    this.ability = ability;
+    this.crew = [];
+  }
+
+  missionStatement(message) {
+    if (this.crew.length) {
+      return this.ability;
+    } else {
+      return "Can't perform a mission yet.";
+    }
+  }
+}
 
```

Comment:
You should make a instance of the crewMember and the Ship,
Then have them enter the ship.